ConstraintsMiguelAskar/ConstraintsMiguelAskar/Assets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This does not belong to this PR.

Reply:
Double checked and this was happening because the PR was targeting `main` whereas it should  target `feature/IA-5/UI-constraint-exercises` which is your base branch for Constraint challenges, updated it and looks fine now.